I just updated my data for VacationPy.ipynb since I noticed an error in spelling. 
